frontend/i18n-scripts/build-i18n.sh

```diff
@@ -6,5 +6,5 @@ i18next "public/!(dist)**/**/*.{js,jsx,ts,tsx}" [-oc] -c "./public/i18next-parse
 
 cd packages
 for d in */ ; do
-  i18next "${d}!(dist)**/**/*.{js,jsx,ts,tsx}" [-oc] -c "./../public/i18next-parser.config.js" -o "${d}locales/\$LOCALE/\$NAMESPACE.json"
+  i18next "${d}!(dist|node_modules)**/**/*.{js,jsx,ts,tsx}" [-oc] -c "./../public/i18next-parser.config.js" -o "${d}locales/\$LOCALE/\$NAMESPACE.json"
```

Comment:
Good point.

`public` is technically a helper/internal package (see `public/package.json`) and for now, `public/node_modules` is absent since all of its dependencies are hoisted up to root `node_modules` directory.

I'll add it on line 5 as well, just to be on safe side.